index.html

```diff
@@ -1034,60 +1034,107 @@
+about people that is shared with or transferred between parties on the Web.</span>
 
-<p><span class="practicelab" id="user-agent-data-sharing">
-User agents should only share and present information about the user that
-the user can reasonably anticipate being shared, and which directly relate to the users overt, immediate goals.
-User agents and sites should be conservative when deciding if information is related to a users goals; the
-longer the chain of reasoning used to justify data collection, the less likely for that data
-collection to be ethical and principled.
-</span></p>
+Data minimization limits the risks of data being disclosed or misused and also makes it possible to
+more meaningfully provide people with understandable decisions about data about them.
 
-</div>
-
-Examples:
+<span class="practicelab">Web APIs should be designed to minimize the amount of data that is requested
+and provide granularity and user controls over personal data that is communicated to sites.</span>
 
-* Sharing geo-location data with a map website after a user has clicked a "share my location" button has
-  a small number of "hops" between intent and data.
-* Sharing information about the user's selected DNS resolver with a site, so that the site authors can debug
-  possible issues around tail/uncommon resolvers, has a large number of "hops" between user intent (to do
-  a thing on the website) and collection motivation (users like our site, so users will want to share information
-  with us to help us improve the site).
-
-<div class="practice">
-
-<p><span class="practicelab" id="sensitive-vs-non-sensitive-data">
-User agents should generally not attempt to distinguish between sensitive and nonsensitive personal data; all
-data and personal information is likely to be sensitive to some users, often in ways unanticipated by
-browser vendors and site operators.
-</span></p>
+<aside class="note">
+This principle was further explored in an earlier TAG draft on <a
+href="https://www.w3.org/2001/tag/doc/APIMinimization-20100605.html">Data Minimization in Web
+APIs</a>.
+</aside>
 
-</div>
+In maintaining duties of <a href="#dfn-duty-of-protection">protection</a>, <a
+href="#dfn-duty-of-discretion">discretion</a> and <a href="#dfn-duty-of-loyalty">loyalty</a>, user
+agents should be conservative when sharing data not necessary to satisfy a user's immediate goals.
+
+On the Web, data is often shared even outside the need to load and display a page for <dfn>ancillary
+uses</dfn> including: 
+
+* browser telemetry, including: 
+  * performance measurements, 
+  * measuring feature usage, 
+  * debugging site, browser and networking issues, and, 
+  * detecting security problems or attacks; 
+* site telemetry, including: 
+  * performance measurements, 
+  * analytics about usage and visitors, 
+  * debugging site issues, and, 
+  * detecting security problems or attacks; 
+* advertising and social media sharing, including: 
```

Comment:
advertising use-case that targets users is quite different from the list of straight-up telemetry use-cases, that don't involve any personal data or device identifiers.
Might make sense to split these?